routes/dashboard/portals/feedbacks-24x7/index.js

```diff
@@ -54,38 +54,41 @@ router.post("/step-4", function(req, res, next) {
         instructor: instructoremail, // Instructor's email
         type: "24x7", // 24x7 or midsem
         responses: feedback,
-        createdOn: Date.now()
+        createdOn: Date.now(),
       };
       feedbacksModel.create(dataStore, function(err, response) {
+        let sendMail = false;
         if (err) {
           res.renderState("custom_errors", {
             redirect: "/dashboard",
             timeout: 5,
             supertitle: "Couldn't submit feedback",
             message: "Failure",
-            details: err
+            details: err,
+          });
+        }
+        if (sendMail) {
```

Comment:
This condition will always be false and thus this is a code path which will never be executed. Having blocks of dead code is not a good practice. 

My recommendation: Have this block controlled by an environment variable / database config.